src/simulation.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+
+from coordinate_system import CoordinateSystem
+from coordinate_system import CartesianCoordinateSystem
+
+
+class Simulation:
+  
+  def __init__(self,
+    rocket_components, # list of rocket components (typically, there will be just one element to start)
+    coordinate_system: CoordinateSystem=CartesianCoordinateSystem, # Coordinate system to use
+    time_max=100, # time that this simulation will cover (affects number of iterations)
```

Comment:
Is this a placeholder value?

Reply:
Safety value that we'll use to make sure we don't infinitely loop.